plugins/outputs/appdynamics/appdynamics.go

```diff
@@ -0,0 +1,127 @@
+package influxdb
+
+import (
+	"encoding/json"
```

Comment:
Can you leave a one-line gap between the std library imports and the third party imports?